README.md

```diff
@@ -2,120 +2,100 @@
     * [Issues and feature request](#issues-and-feature-request)
 
-### Project description
+## Project description
 
-The aim of this project is to show the real-time capabilities using the ROS2 framework. This project is based on the [previous](https://index.ros.org/doc/ros2/Tutorials/Real-Time-Programming/) work made by the Open Robotics. The project uses these packages as a base.
+The aim of this project is to show the real-time capabilities using the ROS2 framework. This
+ project is based on the [previous](https://index.ros.org/doc/ros2/Tutorials/Real-Time-Programming/)
+  work made by the Open Robotics. The project uses these packages as a base.
 
 This is the motivation of using an inverted pendulum for a real-time demo:
 
 >A classic example of a controls problem commonly solved by real-time computing is balancing an inverted pendulum. If the controller blocked for an unexpectedly long amount of time, the pendulum would fall down or go unstable. But if the controller reliably updates at a rate faster than the motor controlling the pendulum can operate, the pendulum will successfully adapt react to sensor data to balance the pendulum.
 
-For a more detailed description of demo the project design be found here: [Design Article](docs/design.md)
+A more detailed description of demo the project design be found here: [Design Article](docs/design.md)
 
-#### Install instructions
+## Install instructions
 
-In order to get the project ready, first you need to install [ROS2](https://index.ros.org/doc/ros2/Installation/Eloquent/) and [rosdep](http://wiki.ros.org/rosdep).
+ Several ways to install the pendulum demo project are provided depending on the user needs. 
+ 
+- [ADE based installation](docs/installation.md#ade#based#installation): This provides a 
+pre-configured Docker image with everything ready to be used. This is the recommended option to
+ quickly test the demo. It is also useful to have development environment to add new features.
+- [Installation from source](docs/installation.md#installation#from#source): provides instructions
+ to build all the required ROS 2 packages from source
+- [Crosscompilation](docs/installation.md#crosscompile): provides instructions to
+ crosscompile the demo for an embedded device
+- [Pre-configured Raspberry PI image](docs/installation.md#raspberry#pi#image): provides a
+ Raspberry PI with everything configured in a Linux Real-time
 
+## How to run the demo
 
-```bash
-source /opt/ros/eloquent/setup.bash
-mkdir -p ~/pendulum_ws/src
-cd ~/pendulum_ws/src
-git clone https://github.com/ros2-realtime-demo/pendulum
-cd ~/pendulum_ws
-sudo rosdep init
-rosdep update
-rosdep install -q -y --from-paths src --ignore-src --rosdistro eloquent
-colcon build --merge-install # OR colcon build --symlink-install
-```
+### Source your workspace
 
-#### How to run the demo
+If installed as an ADE image:
 
-Terminal 1:
 ```bash
-ros2 launch pendulum_bringup pendulum_bringup.launch.py
+source /opt/ros/foxy/setup.bash
```

Comment:
This will change with the upcoming work.

Reply:
Right, if we merge this first, the instructions should be consistent with the current status. The instructions should be changed accordingly with the upcoming work.

Reply:
I'll include a file on the volume that source it.